src/askui/tools/askui/askui_controller.py

```diff
@@ -200,7 +200,7 @@ def stop(self, force: bool = False) -> None:
             force (bool, optional): Whether to forcefully terminate the process.
                 Defaults to `False`.
         """
-        if not hasattr(self, "process") or self._process is None:
+        if not hasattr(self, "_process") or self._process is None:

```

Comment:
The `hasattr` check seems to be redundant to me because _process is always set in __init__. Do you see a reason to keep it?

```suggestion
        if self._process is None:

```

Reply:
Yep, 100% agree